selfdrive/car/hyundai/enable_radar_tracks.py

```diff
@@ -0,0 +1,43 @@
+import time
+from selfdrive.car.isotp_parallel_query import IsoTpParallelQuery
+
+def _fetch_rdr_fw(CP):
+    return next((fw for fw in CP.carFw if fw.ecu == "fwdRadar"), None)
+
+def _verify_radar(rdr_fw, logcan, sendcan, debug):
+    return IsoTpParallelQuery(sendcan, logcan, 0, [rdr_fw.address], [b'\x10\x07'], [b'\x50\x07'], debug=debug)
+
+def _radar_tracks_enable_query(rdr_fw, logcan, sendcan, debug):
+    
+        new_config = b"\x00\x00\x00\x01\x00\x01"
+        dataId = b'\x01\x42'
+        WRITE_DAT_REQUEST = b'\x2e'
+        WRITE_DAT_RESPONSE = b'\x68'
+        query = IsoTpParallelQuery(sendcan, logcan, 0, [rdr_fw.address],
+                                   [WRITE_DAT_REQUEST+dataId+new_config], [WRITE_DAT_RESPONSE], debug=debug)
+        query.get_data(0)
+
+def _enable_radar_tracks(CP, rdr_fw, logcan, sendcan, debug):
+    try:
+        for retries in range(10):
+            query = _verify_radar(rdr_fw, logcan, sendcan, debug)
+            print(f"radar_tracks: ecu write data by id try {retries+1} ...")
+            if query.get_data(0.1):  # Check if any data is returned
+              _radar_tracks_enable_query(rdr_fw, logcan, sendcan, debug)
+            CP.radarUnavailable = False
+            break
+    except Exception as e:
+        time.sleep(3)
+        print(f"radar_tracks: Failed {retries} try: {e}")
+
+def enable_radar_tracks(CP, logcan, sendcan, debug=False):
+    print("radar_tracks: Try to enable radar tracks")
+
+    rdr_fw = _fetch_rdr_fw(CP)
```

Comment:
```suggestion
    rdr_fw = _fetch_rdr_fw(CP)
```
Is this check necessary?

Reply:
you mean the whole _fetch_rdr_fw query:
`rdr_fw = _fetch_rdr_fw(CP)`
or 
`if rdr_fw is not None:`
?
The fetch query as I understand is needed to get the rdr_fw.address for the other queries. I'm not sure if it's possible for it to not be returned, but I guess it won't hurt to check?

This initial version of the enable_radar_tracks code is tribal knowledge from discord. Initially written for a different model, not the santa fe and was directly in controlsd.py.  Then two other guys improved upon it for their santa fes, and my version is a refactor of their work and extracting it into a separate file. So we're both reverse engineering here at this point :-)

BTW I need to find the original creator's messages to see what car he had, we might be able to support more than just the santa fe.

Reply:
ok, found the message from the guy who most likely came up with this approach in the first place:
https://discord.com/channels/469524606043160576/890005897462558790/894813823213633547
He had/has a Kia Niro EV.

Reply:
I see. I'm more keen to retain similar radar firmware checks from the [debug script](https://github.com/sunnyhaibin/sunnypilot/blob/master/selfdrive/debug/hyundai_enable_radar_points.py):
- We check a list of the supported radar firmware that were verified by the users that radar tracks can be enabled
  - If the radar firmware is in the allow list, we then attempts to enable radar tracks per the method in this PR
  - If the radar firmware is not in the allow list, we return similar debug message in console that radar tracks cannot be enabled

I'm thinking of introducing a label in "SP - Vehicles" under the "Hyundai/Kia/Genesis" section with the status of the radar tracks availability. I will test it out once this PR is merged.